apps/src/code-studio/peer_reviews/PeerReviewLinkSection.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+
+class PeerReviewLinkSection extends React.Component {
+  static propTypes = {
+    submissions: PropTypes.arrayOf(PropTypes.array).isRequired,
+    escalatedSubmissionId: PropTypes.number,
```

Comment:
As you mentioned, this would be a little clearer if it were called `escalatedReviewId`